src/scss/_objects.accordion.scss

```diff
@@ -0,0 +1,17 @@
+.accordion {
+  background-color: #f6f6f6;
```

Comment:
Just wondering, is there a reason why the background color is specified for the accordion? Should it be configurable?